llama_index/llms/base.py

```diff
@@ -25,7 +25,7 @@ class ChatMessage(BaseModel):
     """Chat message."""
 
     role: MessageRole = MessageRole.USER
-    content: Optional[str] = ""
+    content: Optional[Any] = ""
```

Comment:
@logan-markewich one thing I change here is for content type.
In order to let GPT4V use same chat msg as other llm, the content for GPT4V could be array not str
If we convert array to str, it will cause issues for encoding local image files. GPT4V API cannot handle the str input for such image encoding